### DISCUSSION TITLE
Upload project to GitHub

### DISCUSSION THREAD

**Discussion by Pratitb** [tailwindlabs/tailwindcss#5978] [Help] [answered]
I have developed a page using Tailwind CSS. I want to upload it to GitHub pages, how do I do it? What about the node modules, all the unused CSS? What do I do with all this?

**Comment by simonswiss** [comment, marked as answer]
Hey!

None of the `node_modules` or development CSS should be in the directory you put on GitHub pages, those are used in development only.

You can find information on how to host a site on GitHub pages here:

https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site

**Reply by Pratitb** [reply]
Thank you @simonswiss

**Comment by ameer-dev-A** [comment]
Hi how you solve a problem with uploading files with tailwind?

**Comment by ameer-dev-A** [comment]
> Hey!
> 
> None of the `node_modules` or development CSS should be in the directory you put on GitHub pages, those are used in development only.
> 
> You can find information on how to host a site on GitHub pages here:
> 
> https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site

Please check my respiratory by name studio I have same problem with uploading files with tailwind

**Reply by wongjn** [reply]
You have a Vite project in `/project` but then you have installed Tailwind in root. Consider [installing and configuring Tailwind inside the Vite project](https://tailwindcss.com/docs/installation/using-vite), not in root.

**Reply by ameer-dev-A** [reply]
What I must do now
Can you help step by step 
I appreciate your help

**Reply by wongjn** [reply]
1. Change working directory to `/project`:
   ```sh
   cd project
   ```
2. Install Tailwind into `/project`:
   ```sh
   npm install tailwindcss @tailwindcss/vite
   ```
3. Run dev server:
   ```sh
   npm run dev
   ```

**Reply by ameer-dev-A** [reply]
And that's all ! 
Do I must more steps?

**Reply by wongjn** [reply]
No, Tailwind should work in your project now.

**Reply by ameer-dev-A** [reply]
Actually it's not work with github 
I am use another way
Thx bro
